src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -76,23 +76,34 @@ public MembershipResults membershipResults(String currentUser, String uid) {
         // The disjoint of basis plus include and exclude: (Basis + Include) - Exclude
         List<String> groupingMembershipPaths = disjoint(parentGroupingPaths(basisIncludeExcludePaths),
                 parentGroupingPaths(excludePaths));
+        // A list of all group paths, in which the uhIdentifier is listed (including curated groupings), so we can find the intersection with curated groupings
+        List<String> groupAndCuratedGroupingsPaths = groupingsService.allGroupPaths(uid);
+        // The list of all curated groupings
+        List<String> curatedGroupingsPaths = groupingsService.curatedGroupings();
+        // Intersect the two lists
+        groupAndCuratedGroupingsPaths.retainAll(curatedGroupingsPaths);
         // Send all the grouping Membership paths to grouper to obtain grouping descriptions.
         List<Group> membershipGroupings = groupPathService.getValidGroupings(groupingMembershipPaths);
         // Get a list of groupings paths of all basis and include groups that have the opt-out attribute.
         List<String> optOutList = groupingsService.optOutEnabledGroupingPaths(parentGroupingPaths(basisAndInclude));
 
-        List<MembershipResult> memberships = createMemberships(membershipGroupings, optOutList);
+        List<MembershipResult> memberships =
+                createMemberships(membershipGroupings, optOutList, groupAndCuratedGroupingsPaths);
         return new MembershipResults(memberships);
     }
 
-    private List<MembershipResult> createMemberships(List<Group> membershipGroupings, List<String> optOutList) {
+    private List<MembershipResult> createMemberships(List<Group> membershipGroupings, List<String> optOutList,
+            List<String> curatedMemberships) {
```

Comment:
Then `curatedMemberships` should be renamed to `curatedGroupingsPaths` so it's clear what should be passed in.